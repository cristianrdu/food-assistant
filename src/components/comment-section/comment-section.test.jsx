import { shallow } from 'enzyme';
import React from 'react';
import CommentSection from './comment-section';

const mockComments = [{
    data:{
        createdAt:'test',  
        content: 'test',
        recipeId: 'test', 
        userName: 'test', 
        userId: 'test' 
    },
    id: 'test'
}]

it('expects to render the CommentSection component', () => {
    expect(shallow(<CommentSection comments={mockComments}/>)).toMatchSnapshot()
})

 