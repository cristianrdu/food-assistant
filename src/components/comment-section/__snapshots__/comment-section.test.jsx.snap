// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expects to render the CommentSection component 1`] = `ShallowWrapper {}`;
